src/main/resources/db/migration/V2__Insert_SxRocket_Data.sql

```diff
@@ -0,0 +1,117 @@
+-- Insert data into sx_dimension_rocket
+INSERT INTO sx_dimension_rocket (id, meters, feet)
```

Comment:
you need to insert the values within the code, after you fetch them from the API